src/legacy/core_plugins/kibana/common/field_formats/types/string.ts

```diff
@@ -27,42 +29,12 @@ const TRANSFORM_OPTIONS = [
   { kind: 'title', text: 'Title Case' },
   { kind: 'short', text: 'Short Dots' },
   { kind: 'base64', text: 'Base64 Decode' },
-  { kind: 'urlparam', text: 'URL Param Decode' }
+  { kind: 'urlparam', text: 'URL Param Decode' },
```

Comment:
@ppisljar  Am I right that these values should be localized?

Reply:
yes, it seems they are used in field format editor and shown to the user.

Reply:
Internationalized.